client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -378,23 +425,30 @@ export default React.createClass( {
 	},
 
 	render() {
-		const classSet = classNames( {
-			'domain-details': true,
-			selected: true
-		} );
+		const needsOnlyGoogleAppsDetails = this.needsOnlyGoogleAppsDetails(),
+			classSet = classNames( {
+				'domain-details': true,
+				selected: true,
+				'only-google-apps-details': needsOnlyGoogleAppsDetails
+			} ),
+			titleOptions = {
+				context: 'Domain contact information page'
+			};
+
+		let title;
+		if ( needsOnlyGoogleAppsDetails ) {
+			title = this.translate( 'G Suite Account Information', titleOptions );
```

Comment:
Alternate string suggestion: 
* `translate( 'Clear account information' )`&mdash; translations: **27**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/gsuite-for-mapped-domains)